src/test/resources/spec/concordion/annotation/Resources.html

```diff
@@ -0,0 +1,218 @@
+		ensure that there are no conflicts.
+	</p>
+    
+    <p>The annotation has the following options:</p>
+	
+	<table class="annotationSummary">
+	<tr>		
+		<th align="left">value</th><td>One or more files to add to the generated specification:
+			<ul>
+				<li>File name starting with '/' is relative to the root folder</li>
+				<li>File not starting with '/' is relative to the class the annotation is applied to</li>
+				<li>Supports wild card characters ("/images/*.png", "/css/*.*")</li>
+				<li>If no file is found an exception will be thrown</li>
+				<li>Will inspect the HTML and remove any links from the head section that contain "concordion.css" or any of the CSS or JavaScript files that have been added</li>
+				<li>CSS and JS files are treated a little differently than other files:
+					<ul>
+						<li>the specifications html will be updated with the correct link (if linked) or content (if embedded)</li>
+						<li>CSS is appended to the end of the head section so that any styling will override the default Concordion styles</li>
+						<li>any existing links to the resource file will be removed</li>
+					</ul>
+					All other file types must be added to the specification at design time by whomever is writing the specification.
+				</li>
+				
+			</ul>
+		</td>
+	</tr>
+	
+	<tr>	
+		<th align="left">insertType</th><td>EMBEDDED or LINKED - applies to css and js files only</td>
+	</tr>
+	
+	<tr>
+		<th align="left">includeDefaultStyling</th><td>remove the default concordion css styling</td>
+	</tr>
+	
+	</table>
+	<br />
+	
+	<!-- ============================================= -->
+    <div class="example">
+        <h3>Example: Single Linked Resource</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( "demo.css" )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+		
+		<p concordion:execute="#result1 = process(#javaFragment)">will include a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span> into the generated specification
+		and leave the <span concordion:assertTrue="isDefaultCssIncluded()">default CSS</span> in the generated specification.</p>
+	</div>
+
+	<!-- ============================================= -->
+    <div class="example">
+        <h3>Example: Single Embedded Resource</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = "demo.css", insertType = InsertType.EMBEDDED )
```

Comment:
Have to have "value = " when using other parameters or get this error: "ResourcesDemoTest.java:8: annotation values must be of the form 'name=value'"